examples/ColormapControl.ipynb

```diff
@@ -0,0 +1,110 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Set up for JupyterLite\n",
+    "try:\n",
+    "    import piplite\n",
+    "    await piplite.install(['ipyleaflet', 'branca'])\n",
+    "except ImportError:\n",
+    "    pass"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipyleaflet\n",
+    "import json\n",
+    "import pandas as pd\n",
+    "from ipywidgets import link, FloatSlider\n",
+    "from branca.colormap import linear\n",
+    "import random\n",
```

Comment:
Panda is used in 'unemployment = pd.read_csv("US_Unemployment_Oct2012.csv")' and the branca importation is also needed here since here we redefined a linear colormap (the same as the default one).
The other unused importation will be removed.

Reply:
Usually when someone adds a comment it is related to the line just above (unless they refer to several lines).
Here I was mentioning `import random` only.

Reply:
Ok sorry for misunderstanding the comment.